tools/dev/run_dev_server.sh

```diff
@@ -257,8 +261,11 @@ case $CONFIG in
         CLUSTER_GOSSIP_BIND_PORT="7106" \
         CLUSTER_DATA_BIND_PORT="7107" \
         CLUSTER_JOIN="localhost:7100" \
+        PROMETHEUS_MONITORING_PORT="2115" \
+	PROMETHEUS_MONITORING_METRIC_NAMESPACE="weaviate" \
```

Comment:
Are these directly parsed from env by the new libraries introduced?
The reason i'm asking is that if they're not we need to introduce handling and default values in our `usecases/config` pkg. 

Also can we make the metric namespace default to `weaviate` ?

Reply:
yea. It's already handled via [config](https://github.com/weaviate/weaviate/blob/main/usecases/monitoring/prometheus.go#L35) and [handled default config](https://github.com/weaviate/weaviate/blob/main/usecases/config/environment.go#L51). I was using it on our querier before. It's just not passed in our local scripts for core-weaviate.